mbuild/utils/conversion.py

```diff
@@ -24,51 +26,63 @@ def RB_to_OPLS(c0, c1, c2, c3, c4, c5, error_tol=1e-4):
         (in kcal/mol).
 
 
     Notes
     -----
     c5 must equal zero, or this conversion is not possible.
 
-    (c0 + c1 + c2 + c3 + c4 + c5) must equal zero, or this conversion is not possible.
+    (c0 + c1 + c2 + c3 + c4 + c5) must equal zero, to have the exact
+    energies represented in the dihedral.
+
+    NOTE: fO IS TYPICALLY NOT IN THE OPLS DIHEDRAL EQUATION AND IS
+    ONLY USED TO TEST IF THIS FUNCTION CAN BE UTILIZED, AND
+    DETERMINE THE fO VALUE FOR LATER ENERGY SCALING IN MOLECULAR DYNAMICS
+    (MD) SIMULATIONS. THIS FUNCTION TESTS IF f0 IS ZERO (f0=0).
 
-    NOTE: fO IS NOT IN THE OPLS DIHEDRAL EQUATION AND IS ONLY USED
-    TO TEST IF THIS FUNCTION CAN BE UTILIZED. ONE OF THE
-    CONDITIONS OF CONVERSION IS f0 MUST BE ZERO (f0=0).
+    WARNING: The f0 term is the constant for the OPLS dihedral equation.
+    If the f0 term is not zero, the dihedral is not an exact conversion;
+    since this constant does not contribute to the force equation,
+    this should provide matching results for MD, but the energy for each
+    dihedral will be shifted by the f0/2 value.
     """
-    if isinstance(error_tol, float):
-        error_tol = abs(error_tol)
-    else:
+    try:
+        error_tol = abs(float(error_tol))
+    except:
         raise TypeError("ERROR: The error_tol variable must be a float.")
 
-    if not np.all(np.isclose(c5, 0, atol=error_tol, rtol=0)) is False:
+    if not np.all(np.isclose(c5, 0, atol=error_tol, rtol=0)):
         raise ValueError(
             "ERROR: c5 must equal zero, so this conversion is not possible."
         )
 
     f0 = 2.0 * (c0 + c1 + c2 + c3 + c4 + c5)
-    if not np.all(np.isclose(f0, 0, atol=error_tol, rtol=0)) is False:
-        raise ValueError(
-            "ERROR: f0 = 2 * (c0 + c1 + c2 + c3 + c4 + c5) must equal zero, "
-            "so this conversion is not possible."
+    if not np.all(np.isclose(f0, 0, atol=error_tol, rtol=0)):
+        warn(
+            "WARNING: f0 = 2 * (c0 + c1 + c2 + c3 + c4 + c5) is not zero. "
+            "The f0 term is the constant for the OPLS dihedral. "
+            "Since the f0 term is not zero, the dihedral is not an exact conversion; "
+            "since this constant does not contribute to the force equation, "
+            "this should provide matching results for MD, but the energy for each "
+            "dihedral will be shifted by the f0/2 value."
         )
 
     f1 = -2 * c1 - (3 * c3) / 2
     f2 = -c2 - c4
     f3 = -c3 / 2
     f4 = -c4 / 4
-    return np.array([f1, f2, f3, f4])
+    return np.array([f0, f1, f2, f3, f4])
 
 
-def OPLS_to_RB(f1, f2, f3, f4):
+def OPLS_to_RB(f0, f1, f2, f3, f4, error_tol=1e-4):
     r"""Convert OPLS type to Ryckaert-Bellemans type dihedrals.
 
     .. math::
-    OPLS_torsions &= f1/2*(1+cos(t)) + f2/2*(1-cos(2*t)) + \\
-                  &= f3/2*(1+cos(3*t)) + f4/2(1-cos(4*t))
+    OPLS_torsions &= f0/2 + f1/2*(1+cos(t)) + f2/2*(1-cos(2*t)) + \\
```

Comment:
```suggestion
    OPLS_torsions &= \frac{f_0}{2} + \frac{f_1}{2}*(1+cos(t)) + f2/2*(1-cos(2*t)) + \\
```

Since this will be rendered with latex math formatting, we should go through and make this string a bit more latex-style since that will make rendering a bit nicer.